libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -870,6 +892,21 @@ bool AC_PolyFence_loader::load_from_storage()
             _num_loaded_circle_inclusion_boundaries++;
             break;
         }
+        case AC_PolyFenceType::HOME_CIRCLE_INCLUSION: {
+            InclusionCircle &circle = _loaded_circle_inclusion_boundary[_num_loaded_circle_inclusion_boundaries];
+            // TODO get the home location here.
+            auto const home = AP::ahrs().get_home();
```

Comment:
SHould not be populating home loc here

Reply:
If we don't populate it here, would you be open to instead changing the _loaded_circle_inclusion_boundary array of `InclusionCircle` to have a boolean to indicate whether they are home-centered? 
Or, perhaps we can encode a NaN lat-lon value as home centered for those, and then have the `breach` checks just query home if lat-lon is NaN in that data structure? I tried not to add more data structures and cases to the switch statements to avoid flash increase.

Reply:
I'm fine with the nan/nan approach; it is not externally facing so we can change it if we feel like it.

An alternative approach would be to subclass InclusionCircle for HomeInclusionCircle - probably only one or two methods would need overriding....

Reply:
I've implemented the NaN approach (but with INT32_MAX); it's working nicely.